src/config.ts

```diff
@@ -164,6 +166,8 @@ export const config = {
   indexSettings,
   indexSynonyms,
   indexRules,
+  expiresAt: 30 * 24 * 3600 * 1000,
+  populareExpiresAt: 7 * 24 * 3600 * 1000,
```

Comment:
these can be configured with `ms`, for consistency with the rest :)

Reply:
good point 👌🏻